qrest/conf.py

```diff
@@ -223,6 +223,7 @@ def __init__(
         processor: Optional[Type[Resource]] = None,
         description: Optional[str] = None,
         path_description: Optional[dict] = None,
+        timeout: Optional[float] = None
```

Comment:
Not sure what to do here, since timeout is allowed to be None, float, int or tuple. Any suggestion?

Reply:
Lets continue this in the other thread.